website/marietje/templates/marietje/index.html

```diff
@@ -0,0 +1,24 @@
+{% extends 'tosti/base.html' %}
+{% load static queue %}
+
+{% block page %}
+    <div class="container-sm">
+        <h1 class="mb-3"><i class="fas fa-music"></i> Marietje</h1>
+        <p class="text-left">Marietje is the music control system for the student canteens in the Huygens building.
```

Comment:
Maybe this text should be less history and more 'what can you do now'

Reply:
Good point, will adjust